spec/onchain.tex

```diff
@@ -414,19 +476,39 @@ \subsection{Close Transaction}\label{sec:close-tx}
 		\[
 			\eta' = \eta
 		\]
-		\item $\mathsf{Current}$: Closing snapshot refers to current state version $v$ and any UTxO to decommit need to be present in the closed state too.
-		\[
-			\msVfy(\hydraKeys,(\cid || v || s' || \eta' || \eta_{\Delta}'),\xi) = \true
-		\]
-		which implies $\eta_{\Delta}' = \eta_{\omega}$ while $\eta_{\omega}$ does not need to be provided.
-		\item $\mathsf{Outdated}$: Closing snapshot refers the previous state $v - 1$ and any UTxO to decommit of the closing snapshot must not be recorded in the closed state.
-		\[
-			\eta_{\Delta}' = \bot
-		\]
-		\[
-			\msVfy(\hydraKeys,(\cid || v - 1 || s' || \eta' || \eta_{\omega}),\xi) = \true
-		\]
-		where $\eta_{\omega}$ is provided by the redeemer\footnote{$\eta_{\omega}$ needs to be provided to verify the signature, but is otherwise not relevant for the closed state}.
+		\red{
+			\[
+			\theta = Normal
+			\]
+			\item $\mathsf{Unused/Inc}$: Closing snapshot refers to current state version $v$, snapshot type $\theta = Inc$ and any UTxO to increment must be $\bot$ in the closed state.
+			\[
+			\msVfy(\hydraKeys,(\cid || v || s' || \eta' || \eta_{\Delta}' || \theta),\xi) = \true
+			\]
+			which implies $\eta_{\Delta}' = \eta_\theta$ while $\eta_\theta$ must not be provided.
+			\item $\mathsf{Unused/Dec}$: Closing snapshot refers to current state version $v$, snapshot type $\theta = Dec$ and any UTxO to decrement need to be present in the closed state too.
+			\[
+        \msVfy(\hydraKeys,(\cid || v || s' || \eta' || \eta_{\Delta}' || \theta),\xi) = \true
+			\]
+			which implies $\eta_{\Delta}' = \eta_\theta$ while $\eta_\theta$ must be provided.
+			% TODO: put more into the redeemer to simplify?
+			\todo{this is hard to understand}
+			\item $\mathsf{Used/Dec}$: Closing snapshot refers the previous state $v - 1$ , snapshot type $\theta = Dec$  and any UTxO to decrement of the closing snapshot must not be recorded in the closed state.
+			\[
+				\eta_{\Delta}' = \bot
+			\]
+			\[
+        \msVfy(\hydraKeys,(\cid || v - 1 || s' || \eta' || \eta_\theta || \theta),\xi) = \true
+			\]
+			where $\eta_\theta$ is provided by the redeemer\footnote{$\eta_\theta$ needs to be provided to verify the signature, but is otherwise not relevant for the closed state}.
+			\item $\mathsf{Used/Inc}$: Closing snapshot refers the previous state $v - 1$ , snapshot type $\theta = Inc$  and any UTxO to increment of the closing snapshot must be recorded in the closed state.
+			\[
+				\eta_{\Delta}' \neq \bot
+			\]
+			\[
+				\msVfy(\hydraKeys,(\cid || v - 1 || s' || \eta' || \eta_\theta || \theta),\xi) = \true
+			\]
+			where $\eta_\theta$ must be provided in the closed state.
```

Comment:
No tonly $\neq \bot$, but $= \eta_\theta$!

```suggestion
			\[
				\eta_{\Delta}' = \eta_\theta
			\]
			\[
				\msVfy(\hydraKeys,(\cid || v - 1 || s' || \eta' || \eta_\theta || \theta),\xi) = \true
			\]
```

This is, in fact, an alternative way to describe the same case as in (c). Implementation wise, the equality $\eta_{\Delta}' = \eta_\theta$ allows us to insert $\eta_{\Delta}'$ directly into $(\mathsf{cid} || v - 1 || s' || \eta_{\Delta}' || \eta_\theta || \theta),\xi)$.. which gives the above definition in (c).

Pick one way of describing it.